packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -270,6 +271,53 @@ export const transformPlotlyJsonToSankeyProps = (jsonObj: any): ISankeyChartProp
   };
 };
 
+export const transformPlotlyJsonToGaugeProps = (jsonObj: any): IGaugeChartProps => {
+  const { data, layout } = jsonObj;
+  const firstData = data[0];
+
+  const segments = firstData.gauge?.steps?.map((step: any, index: number): IGaugeChartSegment => {
+    return {
+      legend: step.name || `Segment ${index + 1}`,
+      size: step.range?.[1] - step.range?.[0],
+      color: step.color || getNextColor(index),
+    };
+  });
+
+  let sublabel: string | undefined;
+  let sublabelColor: string | undefined;
+  if (typeof firstData.delta?.reference === 'number') {
+    const diff = firstData.value - firstData.delta.reference;
+    if (diff >= 0) {
+      sublabel = `\u2BC5 ${diff}`;
```

Comment:
what does this unicode represent

Reply:
this is for the up-pointing triangle when delta is positive